docs/src/api/params.md

```diff
@@ -164,6 +165,20 @@ Specify environment variables that will be visible to the browser. Defaults to `
 Populates context with given storage state. This method can be used to initialize context with logged-in information
 obtained via [`method: BrowserContext.storageState`]. Either a path to the file with saved storage, or an object with the following fields:
 
+## csharp-java-context-option-storage-state
+* langs: csharp, java
+- `storageState` <[string]>
+
+Populates context with given storage state. This method can be used to initialize context with logged-in information
```

Comment:
This method -> this option. Please also change the js version 😄

Reply:
Nice catch!